tests/installation/releasenotes.pm

```diff
@@ -65,6 +65,9 @@ sub run {
         @no_relnotes = grep(!/^we$/, @no_relnotes);
         # HA-GEO has been removed on SLE 15
         @addons = grep(!/^geo$/, @addons);
+        # SES6 does not have RN now bsc#1090005
+        push @no_relnotes, qw(ses);
+        record_soft_failure 'bsc#1090005 - missing SES6 release notes' if check_var('SCC_ADDONS', 'ses');
```

Comment:
No need for qw here and it might be cleaner to put it all inside if:
```
if (check_var('SCC_ADDONS', 'ses')) {
    push @no_relnotes, 'ses';
    record_soft_failure 'bsc#1090005 - missing SES6 release notes'
}

Reply:
ok you miss one ; :)